### DISCUSSION TITLE
Ability to change the Email Text to make it more understandable

### DISCUSSION THREAD

**Discussion by waqadArshad** [saran-surya/email_auth_node#17] [Q&A] [answered]
I think we should also have the option to change the Email Text to make it more understandable for the user and more relatable to the situation.

**Comment by saran-surya** [comment, marked as answer]
Hi @waqadArshad , kindly go through the documentation, you always have a index.html file to modify, you have to modify the file, then use the ```email-auth-node --generate``` to generate the config. Just make sure the values wrapped in {} are still intact after modification.
Also kindly tell me which part of the text is confusing, it is pretty straight forward right ??

**Reply by waqadArshad** [reply]
yes, thanks a lot. I don't know how I missed that. and by making it more understandable I meant to make it more user-friendly. like if I am using it for password reset then I would have to mention that. or just get the whole text from the app to make it more related to the circumstances it is being used in.

**Reply by saran-surya** [reply]
Okay so now I get your concern, will be adding it as a feature request to change the subject of the mail also, I think that should be fine. Please give me some time and I will be implementing them.

Thanks ✌🏼

**Reply by waqadArshad** [reply]
Sure, Thanks!
